__tests__/api/app.test.js

```diff
@@ -252,5 +253,72 @@ describe('Testing app', () => {
           });
       });
     });
+
+    describe('Method POST', () => {
+      test('GET: 201 status responds wih the posted comment', () => {
+        return request(app)
+          .post('/api/articles/2/comments')
+          .send({ username: 'rogersop', body: 'comment test' })
+          .expect(201);
+      });
+
+      test('GET: 201 status', () => {
```

Comment:
I think the test descriptions might want to be swapped between the first and second test

Reply:
done